cmd/explorer/main.go

```diff
@@ -96,6 +96,7 @@ func main() {
 		router.HandleFunc("/validators/data/active", handlers.ValidatorsDataActive).Methods("GET")
 		router.HandleFunc("/validators/data/ejected", handlers.ValidatorsDataEjected).Methods("GET")
 		router.HandleFunc("/search", handlers.Search).Methods("POST")
+		router.HandleFunc("/searchAhead/{type}/{search}", handlers.SearchAhead).Methods("GET")
```

Comment:
I don't believe we need a dedicated enpoint for the new search. You should be able to use the old /search endpoint with the new parameters.

Reply:
I am now doing:
```golang
router.HandleFunc("/search", handlers.Search).Methods("POST")
router.HandleFunc("/search/{type}/{search}", handlers.SearchAhead).Methods("GET")
```